Sources/PluginLibrary/swift-options.pb.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 import SwiftProtobuf
 
 
-let Google_Protobuf_FileOptions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
+let Extensions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
```

Comment:
I implemented full JSON support for all of proto2 (including extensions, groups, etc) just to see what it would require.  Turns out to be pretty straightforward, actually.  We should probably rip that back out again to avoid confusion.